Fix assets URL resolution problem

Lailouezzz opened the pull request:
Vite cannot resolve assets dependencies at runtime, causing in production many dependencies located in assets were not included in the static distribution. The assets folder is used by vue for everything that is staticly included like some json config files, some webp etc

Lailouezzz committed:
Assets are now loaded by relative URL
	- Modified: loadAssets return an universal relative URL

Dyamen1411 committed:
Removed merge conflict...

Removed:
	- There were a merge conflict header
		hiding within `utils/index.js`...

Dyamen1411 reviewed (approved):
Great catch, will merge (beware of leftover merge conflicts...)

Lailouezzz committed:
Major assets fix

Modified: some assets directories are moved to
	public/ressources to avoid vite alias resolution
	problem (the ressources was not loaded in prod).

Lailouezzz committed:
Fix: rebuild nocache now remove the volumes and build no cache
